### PR TITLE
FLUID-5686: Added migration docs

### PR DESCRIPTION
**Pull request by jobara**
Added migration docs which mention the change from emphasize links and inputs larger to enhance inputs.

https://issues.fluidproject.org/browse/FLUID-5686

Related PR: https://github.com/fluid-project/infusion/pull/815

### CODE REVIEW

**Comment by cindyli** [comment]
The doc change in this pull request seems corresponding to adding the switch component instead of collapsing emphasize links and inputs larger into enhance links. @jobara, can you please double check? Thanks.

**Comment by jobara** [comment]
@cindyli I've updated with master, so the changes should be easier to see now. They are fairly minor.

File: src/documents/APIChangesFrom2_0To3_0.md [hunk 1]
```diff
@@ -17,10 +17,17 @@ This section describes major APIs that were in common use. For information about
 
 #### Panel Changes ####
 
+"Links and Buttons" adjuster and enactors collapsed to a single preference, "Enhance Links"
+
 ##### Message Bundle Keys #####
 
 ###### Additions ######
 
+* enhanceInputs.json
```
Comment (cindyli) [inline src/documents/APIChangesFrom2_0To3_0.md]: Would be nice to mention the removal of `inputsLarger.json`, `emphasizeLinks.json`, "linksControls.json`.

File: src/documents/APIChangesFrom2_0To3_0.md [hunk 2]
```diff
@@ -17,10 +17,17 @@ This section describes major APIs that were in common use. For information about
 
 #### Panel Changes ####
 
+"Links and Buttons" adjuster and enactors collapsed to a single preference, "Enhance Links"
```
Comment (cindyli) [inline src/documents/APIChangesFrom2_0To3_0.md]: "... and enactors are collapsed... ", missing "are". Also, might be better to replace the comma with a dash or a colon?

**Comment by jobara** [comment]
@cindyli ready for more review

**Comment by cindyli** [comment]
Merged at 2950c5bed58bad32d670fa723d6625862bbd85dc

**Pull request merged**